Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -394,6 +380,23 @@ def bump_formula_pr
     end
   end
 
+  def forked_repo_info(tap_full_name)
+    response = GitHub.create_fork(tap_full_name)
+  rescue GitHub::AuthenticationFailedError, *GitHub.api_errors => e
+    formula.path.atomic_write(backup_file) unless args.dry_run?
```

Comment:
`args.dry_run` will never be `true` here.

Reply:
ah, nice catch!